_posts/2020-12-24-newpipe-0.20.7.md

```diff
@@ -0,0 +1,60 @@
+
+<!-- more -->
+
+## New in this version
+YouTube Mixes are a thing now, and they are dropping in hot (and with a lot of bass). Opening a mix in previous versions resulted in just the first video of the mix playing, nothing else. Now you see the full playlist and even a nice radio icon as an indicator. Enjoy listening to never-ending playlists and thank [@XiangRongLin](https://github.com/XiangRongLin) for this feature, as well as [@Stypox](https://github.com/Stypox) for reviewing it thoroughly.
+
+This release includes support for another feature from YouTube: video metadata. The little boxes that appear sometimes - telling you that a channel belongs to a public broadcaster, or giving you further information regarding specific search queries - appear in NewPipe as well now. If you do not want to see them, head over to <code><nobr><img src="{{ site.baseurl }}/img/icons/baseline-settings-20px.svg" /> Settings</nobr> > <nobr><img src="{{ site.baseurl }}/img/icons/baseline-language-20px.svg" /> Content</nobr> > <nobr>Show meta info</nobr></code> and disable the toggle there. If you enjoy these boxes however, thank [@TobiGr](https://github.com/TobiGr) for baking them into this release.
+
+On a somewhat more personal note, we have a new domain now - [https://newpipe.net](https://newpipe.net). This makes us look more professional (we hope), but also means that the website URLs in the app needed to be updated. This was done by [@TiA4f8R](https://github.com/TiA4f8R).
+
+## Bugs, be gone!
+We aim for the best version of NewPipe with every release we do. Because of that, we improve on issues in every update. This time around, we fixed an issue where the app would crash when you tried to delete a video from your playlist while the app was refreshing your subscriptions. This was discovered by [@gasull](https://github.com/gasull), reproduced by [@chilliger](https://github.com/chilliger) and fixed by [@hlloreda](https://github.com/hlloreda). Big thanks to all three of them.
+
+In another attempt to improve the ReCaptcha experience, [@Stypox](https://github.com/Stypox) fixed a situation where only a blank page was shown. They also prevented device information from being leaked inside the user agent used for the ReCaptcha service, and are likely very exhausted with that topic at this point. We thank them massively for taking up this issue time and time again.
+
+Talking about time - when sharing video URLs from services other than YouTube, the app doesn't try to append a timestamp to the URL anymore, which used to make them invalid. In the future, we hope to support the timestamp formats of every platform, but for now, you get non-timestamped URLs that work. This was provided by [@B0pol](https://github.com/B0pol). Thank you!
```

Comment:
```suggestion
Talking about time - when sharing video URLs from services other than YouTube, the app doesn't try to append a timestamp to the URL anymore, which makes them invalid. In the future, we hope to support the timestamp formats of every platform, but for now, you get non-timestamped URLs that work. This was provided by [@B0pol](https://github.com/B0pol). Thank you!
```

Reply:
Now it sounds like the new behaviour makes them invalid.

Reply:
I'd also keep the current version.